.changeset/four-mammals-report.md

```diff
@@ -0,0 +1,9 @@
+---
+"create-cloudflare": patch
```

Comment:
this could almost be considered a `minor` since it adds the possibility to customize the app more... I'm happy with either `patch` or `minor` 🙂